packages/config/config/devices/0x027a/zen23_4.0.json

```diff
@@ -0,0 +1,238 @@
+// Zooz ZEN23
+// Z-Wave Plus On/Off Toggle Switch v4
+{
+	"revision": 1,
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN23",
+	"description": "Z-Wave Plus On/Off Toggle Switch v4",
+	"devices": [
+		{
+			"productType": "0xb111",
+			"productId": "0x251c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "4.0",
+		"max": "4.255"
```

Comment:
Unless there's also a 5.x config file, this should be 255.255, so we apply this to all future firmwares until we have a better config file for them.